app/_src/gateway-operator/get-started/kic/create-route.md

```diff
@@ -0,0 +1,70 @@
+---
+title: Create a Route
+content-type: tutorial
+book: kgo-kic-get-started
+chapter: 3
+alpha: true
+---
+
+Now that you've installed all of the required components and configured a `GatewayClass`, it's time to route some traffic to a service in your Kubernetes cluster.
+
+## Configure the echo service
+
+In order to route a request using {{ site.base_gateway }} we need a service running in our cluster. Install an `echo` service using the following command:
+
+```bash
+kubectl apply -f {{site.links.web}}/assets/kubernetes-ingress-controller/examples/echo-service.yaml
+```
+
+## Create a HTTPRoute
+
+Now that we have a service to proxy requests to, we can create a `HTTPRoute` to send any requests that start with `/echo` to the echo service:
```

Comment:
1.  Create a `HTTPRoute` to send any requests that start with `/echo` to the echo service.
```